modules/cluster-logging-release-notes-5.2.z.adoc

```diff
@@ -1,4 +1,50 @@
 //Z-stream Release Notes by Version
+[id="cluster-logging-release-notes-5-2-8"]
+= OpenShift Logging 5.2.8
+
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0728[RHSA-2022:0728 OpenShift Logging Bug Fix Release 5.2.8]
+
+[id="openshift-logging-5-2-8-bug-fixes"]
+== Bug fixes
+* Before this update, the logging link in OpenShift WebConsole remained after the removal of an instance of OpenShift Logging. With this update, the removal or uninstallation of logging also removes the link. (link:https://issues.redhat.com/browse/LOG-2180[LOG-2180])
+
+[id="openshift-logging-5-2-8-CVEs"]
+== CVEs
+.Click to expand CVEs
+[%collapsible]
+====
+* link:https://access.redhat.com/security/cve/CVE-2020-28491[CVE-2020-28491]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=1930423[BZ-1930423]
+* link:https://access.redhat.com/security/cve/CVE-2022-0552[CVE-2022-0552]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=2052539[BG-2052539]
+====
+
+[id="cluster-logging-release-notes-5-2-7"]
+= OpenShift Logging 5.2.7
+
+This release includes link:https://access.redhat.com/errata/RHBA-2022:0478[RHBA-2022:0478 OpenShift Logging Bug Fix Release 5.2.7]
+
+[id="openshift-logging-5-2-7-bug-fixes"]
+== Bug fixes
+* Before this update, Elasticsearch pods failed to start after updating to OpenShift-logging 5.2.3-31 with FIPS enabled. With this update, Elasticsearch pods start successfully. (link:https://issues.redhat.com/browse/LOG-2000[LOG-2000])
+
+* Before this update, elasticsearch generated the error "Unable to create PersistentVolumeClaim due to forbidden: exceeded quota: infra-storage-quota." if the PVC already existed. With this update, elasticsearch checks for existing PVCs, resolving the issue. (link:https://issues.redhat.com/browse/LOG-2118[LOG-2118])
+
+[id="openshift-logging-5-2-7-CVEs"]
+== CVEs
+.Click to expand CVEs
+[%collapsible]
+====
+* link:https://access.redhat.com/security/cve/CVE-2021-3521[CVE-2021-3521]
+* link:https://access.redhat.com/security/cve/CVE-2021-3872[CVE-2021-3872]
```

Comment:
![image](https://user-images.githubusercontent.com/78564304/159023519-8ae16f00-7117-4de4-add8-7e397cde14e3.png)

Reply:
Might have been a localized browser issue, I checked the links in a browser not logged in with any RH creds & they worked.